inc/app_replacer.php

```diff
@@ -142,6 +160,8 @@ abstract class Optml_App_Replacer {
 	 */
 	protected $active_cache_buster_assets = '';
 
+	const DIRECT_UPLOAD_SOURCE = 'directUpload';
```

Comment:
This seems redundant as is not used anywhere. I think it is safe to remove if I'm not missing something.

Reply:
@GrigoreMihai, Thanks for mentioning this! Indeed, it's a leftover from testing! 👍🏻 I'll remove it.